File: qualifier-week-4/graded-assignments/GrPA7 - Graded.lua
local numDays = tonumber(io.read())

if numDays <= 0 then
    return
end

local totalCookies = 0

for _ = 1, numDays do
    local dailySales = tonumber(io.read())
    totalCookies = totalCookies + dailySales
end

print(totalCookies)

File: qualifier-week-4/graded-assignments/GrPA7 - Graded.go
package main

import "fmt"

func main() {
	var numDays int
	fmt.Scan(&numDays)

	if numDays <= 0 {
		return
	}

	totalCookies := 0
	var dailySales int

	for i := 0; i < numDays; i++ {
		fmt.Scan(&dailySales)
		totalCookies += dailySales
	}
	println(totalCookies)
}
